src/sql/workbench/contrib/notebook/browser/media/notebook.contribution.css

```diff
@@ -12,3 +12,14 @@
 .monaco-workbench .activitybar .monaco-action-bar .checked .action-label.book {
 	background-color: rgb(255, 255, 255);
 }
+
+.notebookExplorer-viewlet > .header {
+	height: 41px;
+	box-sizing: border-box;
+	padding: 5px 12px 6px 16px;
+}
+
+.search-view .resultmessages {
```

Comment:
scope this down - we don't want to be modifying the normal search view if they ever decide to add an element with the resultmessages class